crates/runtime/src/flight.rs

```diff
@@ -66,13 +69,16 @@ impl FlightService for Service {
         &self,
         request: Request<FlightDescriptor>,
     ) -> Result<Response<FlightInfo>, Status> {
+        measure_scope_ms!("flight_get_flight_info_request_duration_ms");
```

Comment:
We want `measure_scope_ms` in this and only this function?

Reply:
Yeah, it doesn't work for any of the functions that return streams because it doesn't actually measure the time it takes to do the work and return the data in the stream, just the time it takes to create the stream object and return that - which is always super quick.

GetFlightInfo is one of the exceptions that just returns an object right away and makes sense to track using the macro.